### ISSUE TITLE
Matching members to games as GMs is fragile

### ISSUE THREAD

**Issue by evoskamp** [ggascoigne/amber#92]
For ACUS this year I had 7 failures to recognize GMs in my scheduling because of different spellings of their name in the user table vs whatever they want to put into the GM names.

I think actually letting people pick against the membership table might be handier, and still let them put whatever text they want.

We could even let them pick against user and generate a member record with a flag, or let people (or gamebook or player scheduler) to that.

Thoughts?

**Comment by ggascoigne** [comment]
I'm actually thinking that it might just be worth limiting it to listing other members only.  Because yes, the current mechanism is super fragile.

**Comment by evoskamp** [comment]
Agreed. At least it gets you authoritatively the GM who submits it. And for multi-GM games they can add it later, or the game book scheduler can.

**Comment by evoskamp** [comment]
It'll also makes it easier, and authoritative, to find GMs for a con for a year.
